hera_cal/io.py

```diff
@@ -500,7 +505,10 @@ def write_cal(fname, gains, freqs, times, flags=None, quality=None, write_file=T
     gain_array = np.empty((Nants_data, Nspws, Nfreqs, Ntimes, Njones), np.complex)
     flag_array = np.empty((Nants_data, Nspws, Nfreqs, Ntimes, Njones), np.bool)
     quality_array = np.empty((Nants_data, Nspws, Nfreqs, Ntimes, Njones), np.float)
+    total_quality_array = np.empty((Nspws, Nfreqs, Ntimes, Njones), np.float)
     for i, p in enumerate(pol_array):
+        if total_qual is not None:
+            total_quality_array[:, :, :, 0] = total_qual[p].T[None, :, :]
```

Comment:
I'm confused about this assignment. Shouldn't the dimension that's always index 0 be the spectral window dimension? Shouldn't it be `total_quality_array[0, :, :, i]`?

Also, I think it's preferable to use `np.newaxis` rather than None (it's just clearer what's going on).

Reply:
yeah you're right, thanks for the catch